auth/realm/token/src/test/java/org/wildfly/security/auth/realm/token/OAuth2TokenSecurityRealmTest.java

```diff
@@ -187,7 +187,7 @@ public void testInErrorTokenIntrospectionEndpoint() throws Exception {
 
         RealmIdentity realmIdentity = securityRealm.getRealmIdentity(new BearerTokenEvidence(tokenBuilder.build().toString()));
 
-        assertFalse(realmIdentity.exists());
+        realmIdentity.exists();
```

Comment:
Just to check, since this test case is expected to throw an exception, do we even get to this line or is the exception expected to be thrown on the previous line?

If the exception gets thrown on the previous line, this `realmIdentity.exists()` check could just be removed.

Reply:
Exception is thrown on this line only which is realmIdentity.exists().

Reply:
Thanks for confirming that @keshav-redhat!